pom.xml

```diff
@@ -321,6 +321,11 @@
                 <artifactId>antlr4-runtime</artifactId>
                 <version>${version.antlr.runtime}</version>
             </dependency>
+            <dependency>
```

Comment:
Let's move this to the `monitoring-dashboard/pom.xml` module since this is only used there

Reply:
Fixed in https://github.com/wso2/product-mi-tooling/pull/329/commits/05d0ab83eac91e980e3de9540105565c601ee04a

Reply:
Shall we define the jsoup version as a property?

Reply:
Fixed in https://github.com/wso2/product-mi-tooling/pull/329/commits/21cc3693c4b3d91d7245ed8e43c33d2cceb95955